### PR TITLE
fix: Fix Endpoint Service not being injected in self object

### PR DESCRIPTION
**Pull request by mtahafarooq**
Fixes # [116](https://github.com/Cloud-CV/EvalAI-ngx/issues/116)

#### Changes proposed in this pull request:

- Added the endpointService inside the constructor of the signup component which fixed the endpointService not being found in the self object.

### CODE REVIEW

**Comment by mtahafarooq** [comment]
This PR fixes [116](https://github.com/Cloud-CV/GSoC-Ideas/issues/23) issue.

File: src/app/components/auth/signup/signup.component.ts
Comment (Shekharrajak) [inline src/app/components/auth/signup/signup.component.ts:51]: I wondered, how previously it was tested and merged.

**Review by Shekharrajak — APPROVED** [review]
Looks good to me.

**Comment by lunayach** [comment]
Nice find. Shall we merge this @Shekharrajak @RishabhJain2018?

**Comment by Shekharrajak** [comment]
Yes! I believe it can be merged, after another approval.

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**
